### PR TITLE
Floor price in daily auctions

### PR DESCRIPTION
**Pull request by rokso**
1.  Changed floor price in dailyAuction  to be 1 wei, issue #25 
2.  Fixed Low Severity issue, Formula priceAtInitialAuction may return zero (pg 12). issue #40
3.  Updated tests

### CODE REVIEW

**Comment by rokso** [comment]
@filcap @patidarmanoj10 please review this PR

**Comment by patidarmanoj10** [comment]
@rokso  One test is failing so lets hold this PR.  @filcap FYI. Please do not merge.

**Comment by filcap** [comment]
@rokso looks like it may be just a rounding issue in the unit test.

**Comment by rokso** [comment]
@patidarmanoj10 @filcap as fil said it was rounding issue. Now we have a floor price of 1 wei, due to this calculations going too low and causing rounding issue and we had high floor price before and that is why we didn't see it in this test case.
I did fix the test, also optimized the test by using our shared test util.

File: timed-tests/proceeds.js [hunk 1]
```diff
@@ -24,178 +24,33 @@
...
-    })
-  })
-
-  it('Should return true indicating auction is running', () => {
-    return new Promise(async (resolve, reject) => {
-      await initContracts(1, MINIMUM_PRICE, STARTING_PRICE, TIME_SCALE)
-      assert.ok(await auctions.isRunning(), 'Auctions should be running')
-      resolve()
-    })
-  })
```
Comment (patidarmanoj10) [inline timed-tests/proceeds.js]: These test cases are removed. I understand these are duplicate and tested in other file.  @rokso  Please confirm.

Comment (rokso) [reply timed-tests/proceeds.js]: Yes, these tests are already present under timed-test/auctions.js

**Pull request merged**